src/cdp/domains/target.zig

```diff
@@ -501,6 +505,11 @@ test "cdp.target: createTarget" {
         // should create a browser context
         const bc = ctx.cdp().browser_context.?;
         try ctx.expectSentEvent("Target.targetCreated", .{ .targetInfo = .{ .url = "about:blank", .title = "about:blank", .attached = false, .type = "page", .canAccessOpener = false, .browserContextId = bc.id, .targetId = bc.target_id.? } }, .{});
+
+        // Even about:blank should set the window and document
+        const page = ctx.cdp().browser_context.?.session.page.?;
+        try std.testing.expect(page.state.document != null);
```

Comment:
There's a `const testing = @import("testing.zig");` in this file.

It's `src/cdp/testing.zig` which includes `src/testing.zig`, which extends `std.testing`. There's no `expect` there, but you can add it if you want.I tend to dislike it because it gives less useful errors, but I agree here it's the only real option.

In testing.zig

```zig
pub const expect = std.testing.expect;
```

in cdp/testing.zig

```zig
pub const expect = base.expect;
```

Or not, I think this code here is fine, just wanted to FYI about the existing testing files.